PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/link/mysql/JdbcUrlUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.link.mysql;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.net.URI;
+import java.util.Objects;
+
+public class JdbcUrlUtils {
+
+    public static String parseHostname(String jdbcUrl) {
```

Comment:
`getHostName`?
by the way, `hostname` and `host` may be different.

you may create a class `JdbcUrl`, to simplify the usage and improve performance a little bit.
```
final JdbcUrl url = JdbcUrl.of(url);  // parse the url here.
url.getHostName(); // no need to parse the url again.
url.getDatabase();
url.getPort();
````